### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by janalkema** [kbentlage/da-redis-management#3]
Hi,
After installing I get multiple errors/warnings:  First after installing Redis in ssh
**error 1:**
#Complete!
mount: can't find /tmp in /etc/fstab
pecl/redis is already installed and is the same as the released version 3.1.6
install failed
mount: can't find /tmp in /etc/fstab
Created symlink from /etc/systemd/system/multi-user.target.wants/redis.service to /usr/lib/systemd/system/redis.service.

**error 2.**
I created local.php. In DirectAdmin, when I create a Redis Instance, It creates two instances, 7001 and 7002. I don´t have any idea whi it creates two instances.

and when deleting an instance, I get thos message:
Redis instance is not deleted, due to an unkown error. Back to list.
but when I go back to the list, the instance is deleted.

Any ideas?

**Comment by kbentlage** [comment]
Hi Jan,

Please provide me the contents of your local.php.

Those errors of /tmp in /etc/fstab is because you don't have mounted /tmp as a seperate filesystem (as we do). I will look into this to add some checks in the install scripts.

**Comment by janalkema** [comment]
Hi Kevin,
Thank you for your answer. Here the content of local.php. Although I removed first the plugin and Redis before install it agian, could it be related? Does the mount: can't find /tmp in /etc/fstab error/missing filesystem causes problems?

```
<?php
/**
 * Created by PhpStorm.
 * User: kevin
 * Date: 12/10/2016
 * Time: 21:13
 */

/**
 * In this file you can override all configurations that are defined in the "main.php" configuration file.
 *
 * For example you can define where the Redis data is stored, and under which user redis runs.
 *
 * Please define this in this "local.php" file because of the "main.php" file can be overwritten by new versions.
 */

return [
    'redis' => [
        'dataDir'   => '/home/redis',
    ],
];
```

**Comment by influenist** [comment]
Hi,

IDEAS regarding the setup script:
1. Check if redis is already installed using RPM.
1. First check if there is mount partition at all? Before remounting.
1. Check if there is already redis.so loaded. Now it will add the config parameter non the less its there already.
1. Touch touch /usr/local/directadmin/plugins/redis_management/data/instances.json and fix permissions. e.g. I needed to touch it and chmod it to 777, which doesn't sound secure?

Furthermore your plugin looks neat.

**Comment by kbentlage** [comment]
Implemented several improvements in version 1.1.1
- Improved redis install.sh script (added several checks)
- Added automatic instance cleanup when user is deleted

#5 Version 1.1.1
